### PR TITLE
Move e2e cluster cleanup work from e2e binary to script

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Now we delete edgenode and cleanup other stuff all by e2e binary, just move them to script, e2e binary should only focus on testing.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: tests/e2e/scripts/execute.sh [hunk 1]
```diff
@@ -27,9 +27,13 @@ which ginkgo &> /dev/null || (
     sudo cp $GOPATH/bin/ginkgo /usr/local/bin/
 )
 
-bash ${curpath}/tests/e2e/scripts/cleanup.sh deployment
-bash ${curpath}/tests/e2e/scripts/cleanup.sh edgesite
-bash ${curpath}/tests/e2e/scripts/cleanup.sh device_crd
+cleanup() {
+    bash ${curpath}/tests/e2e/scripts/cleanup.sh deployment
+    bash ${curpath}/tests/e2e/scripts/cleanup.sh edgesite
```
Comment (daixiang0) [inline tests/e2e/scripts/execute.sh:32]: still need it?

File: tests/e2e/scripts/execute.sh [hunk 2]
```diff
@@ -60,6 +64,8 @@ fail=`grep -e "SUCCESS\!" -e "FAIL\!" /tmp/testcase.log | awk '{print $6}' | sed
 echo "Number of Test cases FAILED = $fail"
 echo "==================Result Summary======================="
 
+cleanup
+
```
Comment (daixiang0) [inline tests/e2e/scripts/execute.sh:68]: Add exit signal catch with `trap cleanup EXIT`?

File: tests/e2e/scripts/execute.sh (hunk 1 above)
Comment (fisherxu) [reply tests/e2e/scripts/execute.sh]: Have removed.

File: tests/e2e/scripts/execute.sh (hunk 2 above)
Comment (fisherxu) [reply tests/e2e/scripts/execute.sh:66]: Done.

**Comment by daixiang0** [comment]
lgtm

**Pull request merged**